xapian-applications/omega/templates/query

```diff
@@ -49,17 +50,18 @@ document.write("<span title=\""+D+" "+T+"\">Modified:<br><b>"+D+"</b></span>");
 // -->
 </script>
 </head>
+
 <body bgcolor="#ffffff">
 <form name=P method=get
 action="$html{$or{$env{SCRIPT_NAME},omega}}" target="_top">
 <center>
```

Comment:
I think it's better to rip this stuff out, then come up with new styling when we add CSS (and if we have to add more hooks for selectors at that point, we can). The existing styling is pretty old school ;-)